test/jdk/java/util/concurrent/tck/ForkJoinPool19Test.java

```diff
@@ -492,4 +492,31 @@ public void testAdaptInterruptible_Callable_toString() {
         }
     }
 
+    /**
+     * Implictly closing a new pool using try-with-resources terminates it
```

Comment:
Typo "Implictly" - twice

Reply:
Tests in this file are not being executed.  I think you need:

--- a/test/jdk/java/util/concurrent/tck/ForkJoinPool19Test.java
+++ b/test/jdk/java/util/concurrent/tck/ForkJoinPool19Test.java
@@ -55,7 +55,7 @@ public class ForkJoinPool19Test extends JSR166TestCase {
     }
 
     public static Test suite() {
-        return new TestSuite(ForkJoinPool8Test.class);
+        return new TestSuite(ForkJoinPool19Test.class);
     }
 
     /**

Reply:
```
--- a/test/jdk/java/util/concurrent/tck/ForkJoinPool19Test.java
+++ b/test/jdk/java/util/concurrent/tck/ForkJoinPool19Test.java
@@ -55,7 +55,7 @@ public class ForkJoinPool19Test extends JSR166TestCase {
     }
 
     public static Test suite() {
-        return new TestSuite(ForkJoinPool8Test.class);
+        return new TestSuite(ForkJoinPool19Test.class);
     }
 
     /**
```

Reply:
testLazySubmit will cause jtreg to start hanging.
If you wait patiently for 1000 seconds, you'll get a stack dump.

Reply:
Thanks; now fixed (and enabled). (The test didn't do what doc said about joined "by a worker")